.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Can we remove this file until we need to set a configuration? And can we do it with the `"prettier"` key in `package.json` instead? Just trying to keep the root as clean as possible.